Lib/fontTools/cffLib.py

```diff
@@ -1731,17 +1743,16 @@ def xmlRead(self, name, attrs, content, parent):
 	((12, 36),	'FDArray',		'number',	None,	FDArrayConverter()),
 	(17,		'CharStrings',		'number',	None,	CharStringsConverter()),
 	(24,		'VarStore',		'number',	None,	VarStoreConverter()),
-	(25,		'maxstack',		'number',	None,	None),
```

Comment:
More specifically, I plan to work in this in about two weeks, when I will also remove the global data references.